symfony/framework-bundle/5.4/manifest.json

```diff
@@ -9,11 +9,13 @@
     },
     "composer-scripts": {
         "cache:clear": "symfony-cmd",
-        "assets:install %PUBLIC_DIR%": "symfony-cmd"
+        "assets:install %PUBLIC_DIR%": "symfony-cmd",
+        "secrets:generate-keys": "symfony-cmd",
```

Comment:
the composer scripts run on each install/update. you don't want to regenerate keys every time

Reply:
accoding to the command doc:

```
  If encryption keys already exist, the command must be called with
  the --rotate option in order to override those keys and re-encrypt
  existing secrets.
  
      bin/console secrets:generate-keys --rotate
```

iiuc it is ok like this, after the first install, other updates will change nothing